Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM lambci/lambda:build-nodejs12.x
 # COPY index.js .
 COPY . .
 
-RUN npm ci
+RUN touch fake.js
```

Comment:
who dis?

Reply:
Well this is just a template repo, but I didn't want it failing the build check